outputs.tf

```diff
@@ -1,41 +1,41 @@
 output "storage_account_id" {
-  value       = join("", azurerm_storage_account.storage.*.id)
+  value       = join("", azurerm_storage_account.storage[*].id)
```

Comment:
Remove join function and replace [*] with [0] wherever applicable. 
Pls do for all outputs.